src/fedml/task.py

```diff
@@ -146,7 +146,7 @@ def train_CNN(net, trainloader, valloader, epochs, learning_rate, device):
 
 
 def testing(net, testloader, device):
-    return evaluate_CNN(net, testloader, device)
+    return testing_CNN(net, testloader, device)
```

Comment:
Small nitpick but I tend to prefer verbs for functions and adjectives for context managers, so we have something like

```python
test(...)
```

or 

```python
with testing(...):
    ...
```

just so it reads more nicely.

Reply:
Hi Will!
I had a question regarding naming. Actually, I wanted to name it test() but then weirdly the pytest suit reads it as its own test, despite the file is not in the test folder. Maybe you have a solution?